File: UnitTest/MainActivity.fs
namespace UnitTest

open System.Reflection

open Android.App
open Android.OS
open Xamarin.Android.NUnitLite

[<Activity (Label = "UnitTest", MainLauncher = true)>]
type MainActivity () =
    inherit TestSuiteActivity ()
    
    override this.OnCreate (bundle) =

        this.AddTest (Assembly.GetExecutingAssembly ());
    
        
        base.OnCreate (bundle)

File: Crossplatform_ssp/Crossplatform_ssp.Android/MainActivity.cs
﻿using System;

using Android.App;
using Android.Content.PM;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;

namespace Crossplatform_ssp.Droid
{
	[Activity (Label = "Crossplatform_ssp", 
        Icon = "@drawable/sspLogo", 
        Theme="@style/MainTheme",
        MainLauncher = false, 
        ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
	public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
	{
		protected override void OnCreate (Bundle bundle)
		{
			TabLayoutResource = Resource.Layout.Tabbar;
			ToolbarResource = Resource.Layout.Toolbar; 

			base.OnCreate (bundle);

			global::Xamarin.Forms.Forms.Init (this, bundle);
			LoadApplication (new Crossplatform_ssp.App ());
		}
	}
}

